common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleIntegrationSuite.java

```diff
@@ -232,6 +240,28 @@ public void testFetchThreeSort() throws Exception {
     exec0Fetch.releaseBuffers();
   }
 
+  @Test
+  public void testFetchOneExtendedSort() throws Exception {
```

Comment:
`testFetchOneSortWithShuffleGenerationId`

Reply:
Thanks, done in 3ec15dc.